src/embedded.ts

```diff
@@ -14,6 +14,8 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+// eslint-disable-next-line no-restricted-imports
+import { EventEmitter } from "events";
```

Comment:
Why not a TypedEventEmitter?

Reply:
I am wondering if `room.updatePendingEvent(event, EventStatus.SENT, response.event_id);` emits sth that can be used here. So that I do not need the emitter at all.

Reply:
@t3chguy I updated it so it uses the `RoomEvent.LocalEchoUpdated` emitted  by room.updatePendingEvent.